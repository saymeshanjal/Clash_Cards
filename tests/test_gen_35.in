100
12 17 5
12 65 1
8 167 2
6 80 1
4 58 1
16 10 5
2 69 2
10 89 4
16 168 5
6 53 2
16 58 1
14 104 1
8 35 4
6 189 1
12 103 1
4 62 1
14 142 1
12 162 2
12 162 5
12 124 5
10 67 4
12 57 5
14 165 5
16 31 5
2 132 5
10 166 2
4 36 1
10 122 1
4 199 3
12 92 4
16 96 4
8 133 1
16 68 3
14 115 1
8 60 4
16 160 2
4 25 5
6 30 2
12 159 1
6 84 3
8 11 2
2 46 5
16 45 3
6 196 1
12 87 1
10 23 1
4 16 3
6 10 5
14 71 4
2 9 1
16 5 2
14 21 4
6 20 1
8 71 2
16 100 2
16 144 5
16 121 1
2 67 4
2 153 5
14 34 2
4 97 2
2 143 5
12 105 3
16 52 4
14 99 1
10 58 5
4 137 3
14 18 5
14 178 1
14 176 5
16 154 4
12 18 4
2 119 4
2 198 2
16 108 3
6 32 2
12 97 4
4 120 4
4 158 2
6 188 1
12 19 5
4 8 3
10 177 2
2 97 2
4 193 3
14 163 2
12 153 2
12 186 2
2 131 5
10 141 1
4 50 5
2 59 4
4 111 3
4 112 5
2 17 5
8 23 4
14 92 2
8 162 5
2 50 4
10 134 5
