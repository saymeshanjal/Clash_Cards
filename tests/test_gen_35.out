11 913 0
462 462 0
20 21 29
10 10 0
3 3 0
99 12771 0
0 1 1
44 133 75
4248 8622 0
5 6 9
6410 6460 0
1716 1716 0
8 42 20
10 10 0
462 462 0
3 3 0
1716 1716 0
286 287 351
219 705 0
219 705 0
44 133 75
219 705 0
987 2445 0
367 12503 0
0 2 0
76 77 99
3 3 0
126 126 0
1 5 0
208 441 275
3809 5406 3655
35 35 0
5448 7422 0
1716 1716 0
8 42 20
4081 4082 4707
0 6 0
5 6 9
462 462 0
6 14 0
2 13 55
0 2 0
3018 9852 0
10 10 0
462 462 0
124 128 0
1 5 0
0 20 0
910 1520 1002
1 1 0
495 4257 8118
44 2948 440
10 10 0
20 21 29
4081 4082 4707
4248 8622 0
6435 6435 0
0 2 0
0 2 0
125 312 2995
1 2 3
0 2 0
430 494 0
497 8095 4278
1716 1716 0
45 207 0
1 5 0
60 3372 0
1716 1716 0
987 2445 0
3809 5406 3655
15 827 82
0 2 0
0 1 1
6307 6563 0
5 6 9
208 441 275
0 5 1
1 2 3
10 10 0
15 909 0
1 5 0
76 77 99
0 1 1
1 5 0
1078 1079 1275
286 287 351
286 287 351
0 2 0
126 126 0
0 6 0
0 2 0
1 5 0
0 6 0
0 2 0
5 42 23
1078 1079 1275
8 62 0
0 2 0
45 207 0
